kratos/includes/model_part.h

```diff
@@ -399,6 +399,29 @@ class KRATOS_API(KRATOS_CORE) ModelPart final
         KRATOS_CATCH("")
     }
 
+    template<class TIteratorType >
+    void AddNodesFromOrderedContainer(TIteratorType nodes_begin,  TIteratorType nodes_end, IndexType ThisIndex = 0)
```

Comment:
ThisIndex is not being used

Reply:
Indeed

Reply:
Also, very minor, but we use camel case for parameters, and in this case they should start with "i" to indicate its an iterator

Reply:
Also input should be in CammelCase format

Reply:
I was copying the interface from AddNodes, that TBH has same issue. It is not used. We can Remove both probably.